File: 2022-1/ALGORITMOS/TDE_07:04/TDE_19:05/ex2.c
#include <stdio.h>

int main(){
    int num;
    do
    {
        printf("Digite um numero:\n");
        scanf("%d%*c",&num);
        if(num>0){
            printf ("o numero é positivo\n");
        }else if(num<0){
             printf("numero negativo\n");
        }
        else{
           break;
        }
    } while (num!=0);
}

File: 2023-1/Laboratório de programação/TDE_03/src/ex2.java
import java.util.Scanner;
/*Escreva um programa em Java que recebe um número
inteiro positivo e imprime se ele é um número primo ou
não. Utilize um loop for para testar se o número é primo.*/
public class ex2 {

    public static void main(String[] args) {
        int numero = 0;
        int cont = 0;
        Scanner input = new Scanner(System.in);
        System.out.print("Digite um número inteiro positivo: ");
        numero = input.nextInt();

        for (int i =1; i <= numero; i ++){
            if(numero % i == 0){
                cont ++;
            }

        }if(cont == 2){
            System.out.println("O número" + " " + numero + "é primo" );
        }else{
            System.out.println("O número" + " " + numero + " não é primo");
        }
    }
}
